apps/web/cypress/tests/auth.spec.ts

```diff
@@ -75,4 +75,26 @@ describe('User Sign-up and Login', function () {
       cy.getByTestId('error-alert-banner').contains('Wrong credentials');
     });
   });
+
+  describe('Logout', function () {
+    beforeEach(function () {
+      cy.task('clearDatabase');
+      cy.seed();
+    });
+
+    it('should logout user when auth token is expired', function () {
```

Comment:
awesome that you did the test here! Nice job

Reply:
Thank you.